File: lionu/src/main/java/com/android/reportx/util/RP.java
package com.android.reportx.util;

import android.content.Context;

public class RP {

    public static boolean debug = false;
    public static String base64SecretKey = "";

    public static void launch(Context context) {
        HttpUtil.postUsage(context, ApplicationProvider.launch_url, 0, base64SecretKey);
    }

    public static void reg(Context context) {
        HttpUtil.postUsage(context, ApplicationProvider.reg_url, 0, base64SecretKey);
    }

    public static void pay(Context context, float amount) {
        HttpUtil.postUsage(context, ApplicationProvider.pay_url, amount, base64SecretKey);
    }
}

File: lionukt/src/main/java/com/digitalevers/lionukt/util/RP.kt
package com.digitalevers.lionukt.util

import android.content.Context
import com.digitalevers.lionukt.util.HttpUtil.postUsage

object RP {
    var debug: Boolean = false

    fun launch(context: Context?) {
        postUsage(context, ApplicationProvider.launch_url, 0f)
    }

    fun reg(context: Context?) {
        postUsage(context, ApplicationProvider.reg_url, 0f)
    }

    fun pay(context: Context?, amount: Float) {
        postUsage(context, ApplicationProvider.pay_url, amount)
    }
}
